python/dgl/global_config.py

```diff
@@ -0,0 +1,30 @@
+
+"""Module for global configuration operators."""
+from ._ffi.function import _init_api
+
+__all__ = ["is_libxsmm_enabled", "use_libxsmm"]
+
+def use_libxsmm(flag):
+    r"""Set whether DGL uses libxsmm at runtime.
+
+    Detailed information about libxsmm can be found here:
+    https://github.com/libxsmm/libxsmm
+
+    Parameters
+    ----------
+    flag : boolean
+        If True, use libxsmm, otherwise not.
+    """
+    SetLibxsmm(flag)
```

Comment:
Our CAPI naming style is `_CAPI_XXX`. Pls change.

Reply:
Fixed